dashboard/config/locales/en.yml

```diff
@@ -713,7 +713,7 @@ en:
     content: 'Content'
     practice: 'Teacher Practices'
   external_links:
-    disclaimer_md: "<strong>Disclaimer: You are now leaving the Code.org domain, to a site that is not operated by Code.org.</strong> This site may have privacy policies that differ from Code.org's.<br/><br/>We have selected these third party resources because we think they will be useful to you. While we are not directly responsible for the content of this website, please <a href='mailto:support@code.org'>contact support</a> if you find the content objectionable, or if the linked site is no longer available."
+    disclaimer_md: "<strong>Disclaimer: You are now leaving the Code.org domain, to a site that is not operated by Code.org.</strong> This site may have privacy policies that differ from Code.org's.<br/><br/>We have selected these third party resources because we think they will be useful to you. While we are not directly responsible for the content of this website, please <a href='https://support.code.org'>contact support</a> if you find the content objectionable, or if the linked site is no longer available."
```

Comment:
No, sorry, I meant that the string here should still have the `%{support_url}` param so they don't see the URL in the string that they translate. Then in the view, we have `support_url: https://support.code.org`.